internal/install/recipes/recipe_repository.go

```diff
@@ -117,9 +117,16 @@ func findMaxMatch(matches []recipeMatch) recipeMatch {
 
 	for _, match := range matches {
 		if result == nil {
-			result = &match
+			result = &recipeMatch{
+				recipe:     match.recipe,
+				matchCount: match.matchCount,
+			}
 		} else {
 			if match.matchCount > result.matchCount {
+				result = &recipeMatch{
```

Comment:
This assignment is unnecessary is we're setting `result` again on line 130, right?

Reply:
yep, good catch.

Reply:
fixed